Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -829,9 +829,9 @@ static void dispatchSyntheticMouseMove(LocalFrame& mainFrame, const WebCore::Flo
         auto* localMainFrame = dynamicDowncast<LocalFrame>(m_page->mainFrame());
         if (!localMainFrame)
             return;
-        auto& mainFrame = *localMainFrame;
+        Ref mainFrame = *localMainFrame;
```

Comment:
If you apply Sihui's suggestion above, you don't need this local variable anymore I think.

Reply:
I think so too.